triton/nonblocking_stream.py

```diff
@@ -10,6 +10,8 @@
 Adapted from https://github.com/rhettg/BlueOx/blob/master/blueox/network.py
 
 """
+from __future__ import unicode_literals
```

Comment:
I'm a bit unclear about the implications of only importing this in one file. What is the scope of `unicode_literals` instead of normal string literals?